draft-rokui-ccamp-actn-wdm-pluggable-modelling.md

```diff
@@ -461,84 +448,55 @@ Coherent pluggable have revolutionized optical networking by offering a powerful
 
 ~~~~
 
  |-----------------------------------------------------------------|
- |  optical-channel  // OTSI channels                              |
- |      configuration // list of R/W plug configuration attributes |
- |           config-attribute-1                                    |
- |           config-attribute-2                                    |
- |           .....                                                 |
- |           config-attribute-m                                    |
+ |  optical-channel  // OTSi channels                              |
  |      pm and states  // list of R/O pm and state attributes      |
  |                     // Note-1: Each pm-attribute might have     |
  |                     //         threshold definitions            |
  |                     // Note-2: For each monitored attributes,   |
- |                     //         one SCTP profile can be assigned |
+ |                     //         one SCTG profile can be assigned |
  |           monitored-attribute-1                                 |
  |           monitored-attribute-2                                 |
  |           .....                                                 |
  |           monitored-attribute-p                                 |
  |-----------------------------------------------------------------|
 
 ~~~~
-{: #figure-plug-config-pm title="Data structure for Coherent pluggable Configuration and PM Attributes"}
-
-{: #plug-pm-definition}
-## Coherent Pluggable Performance Monitoring Data
-
-{{figure-plug-config-pm}} shows the list of pluggable Performance Monitoring (PM) and state data, which are critical components in optical networks, enabling network engineers to ensure optimal performance, identify issues, and maintain network reliability. Operators monitor a range of attributes on both the optical/photonic and electrical sides of coherent pluggables, including channel input power, channel output power, central frequency, current Optical Signal-to-Noise Ratio (OSNR), Bit Error Rate (BER), chromatic dispersion, laser temperature, link status, and more. These parameters directly impact the quality and integrity of the transmitted data across both optical and electrical domains.
+{: #figure-plug-pm title="Data structure for Coherent pluggable PM Attributes"}
```

Comment:
line 499:

{: #figure-plug-pm title="Data structure for Coherent module PM Attributes"}